origin.html

```diff
@@ -33,5 +31,64 @@ <h2>Let's get in touch...!</h2>
             <li><a href=""><img src="./images/medium.svg" alt="mediumlogo"></a></li>
         </ul>
     </section>
+    <section class="storysection">
+        <h2 class="storytitle">My Recent Work</h2>
+        <ul class="storyimages">
+            <li><img class="underline" src="./images/story/Vector.png" alt=""></li>
+            <li><a href=""><img class="secondmain" src="./images/SecondPage/ImgPlaceholder.png" alt=""></a></li>
+        </ul>
+        <h3 class="storysubtitle">Multi-Post Stories</h3>
+        <p class="storyparragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="storyimages2">
+            <li class="img1"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag.png" alt="css"></a></li>
+            <li class="img2"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag1.png" alt=""></a></li>
+            <li class="img3"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag2.png" alt=""></a></li>
+            <li class="img4"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag3.png" alt=""></a></li>
+        </ul>
+        <p><button class="second">See Project</button></p>
+    </section>
+
+    <section class="projectsection">
+        <h2 class="projecttitle">Professional Art Priting Data</h2>
+        <p class="projectparagraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="skill">
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag0.png" alt=""></a></li>
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag1.png" alt=""></a></li>
+            <li><a href=""><img class="skill-image-tag" src="./images/project/Tag2.png" alt=""></a></li>
+        </ul>
+        <button class="projectbutton">See Project</button>
+    </section>
+
+    <section class="aboutmesection">
+        <h2 class="aboutmetitle">About me</h2>
+        <p class="aboutmeparragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p><button class="aboutmebutton">Get My Resume</button></p>
+    </section>
```

Comment:
- [ ] You did a good job implementing the sections in your project, but it would be nice to wrap all the related content inside just one section and use `div`'s to wrap all the child elements. For instance, your `About` could look like
```
<section class="about-section">
  <div>
    Content related to about section
  </div>
  <div>
    More content related to about section
  </div>
</section>
```
So please, instead of creating multiple `section` tags, wrap all the related content inside a parent `section` tag and use `div`'s instead.

Reply:
@Lordkaito the changes are already main, please check it again.